libraries/provider_td_rubygems.rb

```diff
@@ -25,6 +25,9 @@ class Package
       class TdRubygems < Chef::Provider::Package::Rubygems
 
         class TdGemEnvironment < AlternateGemEnvironment
+          def rubygems_version
```

Comment:
I don't get your intention to override the inherited `rubygems_version` here 🤔 

https://github.com/chef/chef/blob/3f35e622fd9fc612bff3e5403e737f012aa9bd66/lib/chef/provider/package/rubygems.rb#L304-L306

Reply:
Because in chef12 doesn't exists (still used by AWS Opsworks service)